torchprime/torch_xla_models/model/deepseek_v3/model.py

```diff
@@ -274,13 +274,13 @@ def _post_state_dict_old_keys(self, state_dict, prefix: str):
     E = self.E
     for e in range(E):
       state_dict[f"{prefix}experts.{e}.gate_proj.weight"] = (
-        self.grouped.W_gate[e].t().contiguous().to(BF16)
+        self.grouped.W_gate[e].t().contiguous().to(FP32)
```

Comment:
was it FP32 in HF original implementation?

I understand that layernorms can be FP32, but MOE weights are largest tensors in the whole model, they probably should be bf16.